src/chocolatey/infrastructure.app/commands/ChocolateyNewCommand.cs

```diff
@@ -27,7 +27,7 @@ namespace chocolatey.infrastructure.app.commands
     using services;
     using templates;
 
-    [CommandFor("new", "generates files necessary for a chocolatey package from a template")]
+    [CommandFor("new", "creates template files for creating a new Chocolatey package")]

```

Comment:
creates a new package, not a template. It generates a new package from a template.

Reply:
it kind of is template(d?) files though. It isn't really a package until the `.nupkg` file is created, right?

And usually the files isn't ready to be used as is without editing them. So IMHO, I think template (or templated) files would be more accurate?

Reply:
I agree with @AdmiringWorm here. This is also in line with what is documented on the site here: https://docs.chocolatey.org/en-us/create/commands/new